bin/cosmos

```diff
@@ -38,7 +38,8 @@ def shell(db_url_or_path_to_sqlite):
     cosmos_app.shell()
 
 
-def run(in_jobs, default_drm, default_queue, restart, max_cores):
+def run(in_jobs, default_drm, default_job_class, default_queue,
+        restart, max_cores):
```

Comment:
We are starting to use job classes in addition to queues at Invitae. I know that _queue_ is a more drm-portable term (if not concept) than _job class_ but there is precedence for having fields that are ignored by some drm's (`local` ignores _queue_, for example).

I suppose an alternative approach would be to combine `default_job_class` and `default_queue` into something like `default_drm_parameters` which would take a dictionary. I imagine if another contributor wanted to add LSF- or slurm-specific parameters to a Task then this might be worth looking at more closely. On the other hand, maybe the time is now...

Reply:
lgtm